scripts/src/admin/services.ts

```diff
@@ -337,20 +336,24 @@ async function orderToHtml(order: Order): Promise<string> {
 }
 
 async function userToHtml(user: User): Promise<string> {
+	const accounts = (await user.getWallets()).all().map(wallet => {
+		return `
+		<a href="${ BLOCKCHAIN.horizon_url}/accounts/${ wallet.address }">${ wallet.address }</a>
```

Comment:
You could use DomParser, much more readable for html chunk of code.

Reply:
the entire admin is written in this awful way, we need to change all of it and go with react